.circleci/config.yml

```diff
@@ -0,0 +1,80 @@
+# This file was generated by the excavator check 'excavator/manage-circleci' as specified in .circleci/template.sh.
+# To request a modification to the general template, file an issue on Excavator.
+# To manually manage the CircleCI configuration for this project, remove the .circleci/template.sh file.
+
+version: 2
+jobs:
+  build:
+    docker:
+      - image: ellisjoe/openjdk-java-8-openssl:0.0.2
```

Comment:
would prefer a repo for this to exist so it's easy to see the source / make tweaks (e.g. https://github.com/iamdanfox/docker-oldgit/blob/develop/Dockerfile)

Reply:
done